config/jobs/autobump/autobump.yaml

```diff
@@ -1,73 +1,56 @@
-      - /root/gpg-signing-key/poiana.asc
-      - "EC9875C7B990D55F3B44D6E45F284448FF941C8F"
-      volumeMounts:
-      - name: service
-        mountPath: /etc/service-account
-        readOnly: true
-      - name: github
-        mountPath: /etc/github-token
-        readOnly: true
-      - name: gpg-signing-key
-        mountPath: /root/gpg-signing-key/
-        readOnly: true
-      env:
-    # autobump.sh args
-      - name: GH_PROXY
-        value: https://api.github.com
-      # GitHub org containing the repo where the Prow config and component files live.
-      - name: GH_ORG
-        value: falcosecurity
-      # GitHub repo where the Prow config and component deployment files live.
-      - name: GH_REPO
-        value: test-infra
-      # Repo relative path of the `plank` component k8s deployment file.
-      - name: PROW_CONTROLLER_MANAGER_FILE
-        value: config/prow/prow-controller-manager.yaml
-    # bump.sh args
-      # Directory containing k8s deployment YAMLs for Prow components.
-      - name: COMPONENT_FILE_DIR
-        value: config/prow/
-      # Repo relative path of the core Prow config file (config.yaml).
-      - name: CONFIG_PATH
-        value: config/config.yaml
-      # Repo relative path of the ProwJob config file or directory.
-      # Omit this if ProwJobs are only defined in config.yaml (or are not configured at all).
-      - name: JOB_CONFIG_PATH
-        value: config/jobs/
-      - name: GOOGLE_APPLICATION_CREDENTIALS
-        value: /etc/service-account/service-account.json
-    volumes:
-    - name: service
-      secret:
-        secretName: gcloud-credentials
-    - name: github
-      secret:
-        # Secret containing a GitHub user access token with `repo` scope for creating PRs.
-        secretName: oauth-token
-    - name: gpg-signing-key
-      secret:
-        secretName: poiana-gpg-signing-key
-        defaultMode: 0400
-    nodeSelector:
-      Archtype: "x86"
+periodics:
+  - cron: "05 15 * * 1"  # Run at 15:05 PST (15:05 UTC) Mon
+    name: ci-test-infra-autobump-prow
+    decorate: true
+    extra_refs:
+    - org: falcosecurity
+      repo: test-infra
+      base_ref: master
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/generic-autobumper::v20230509-5e039b951c
+        command:
+        - generic-autobumper
+        args:
+        - --config=config/autobump-config/prow-autobump-config.yaml
+        volumeMounts:
+        - name: github
+          mountPath: /etc/github-token
+          readOnly: true
+        - name: ssh
+          mountPath: /root/.ssh
+      volumes:
+      - name: github
+        secret:
+          secretName: oauth-token
+      - name: ssh
+        secret:
+          secretName: poiana-ssh-keys
+          defaultMode: 0400
+      nodeSelector:
+        Archtype: "x86"
+    annotations:
+      description: runs autobumper to create/update a PR that bumps prow to the latest RC without label 'skip-review'
+
+postsubmits:

```

Comment:
I don't think we need to redeploy prow through this (we already have a circleci job, soon to be gha workflow, for that).
@maxgio92

Reply:
Fixed ;)